### PR TITLE
DOCS-2116: Update board config builders R2D2

### PR DESCRIPTION
**Pull request by sguequierre**
* Update board config builders (except for pca9685 bc attribute is not marked required and card will change)

-  Add analogs attributes to genericlinux boards: confirmed with alan that this was not R2D2 error but was indeed added last July (same implementation as for pi where you plug ADC into board) but has been unused and he wants to remove it but needs to discuss with team? so might revert this (update: did revert this after Alan discussed with rand. May re-document depending on what Eliot thinks)

### CODE REVIEW

File: docs/components/board/beaglebone.md [hunk 1]
```diff
@@ -29,8 +29,6 @@ Enter a name for your board and click **Create**.
 
 ![An example configuration for a beaglebone board in the Viam app Config Builder.](/components/board/beaglebone-ui-config.png)
```
[the remark this answers is not in the document]
Comment (sguequierre) [reply docs/components/board/beaglebone.md:31]: re. analogs-- see pr description, Alan/Rand don't want me documenting that right now

[the remark this answers is not in the document]
Comment (sguequierre) [reply docs/components/board/beaglebone.md:31]: Adding JSON bits + the same config instruction as Pi which is `Edit the attributes as applicable to your board, according to the table below.` Digital interrupts already have the documentation instructions and include below. LMK if I should have more specific instructions (I think change would be made on each of these?)

File: docs/components/board/upboard.md [hunk 2]
```diff
@@ -83,10 +51,7 @@ Then remove and fill in the attributes as applicable to your board, according to
       "namespace": "rdk",
       "attributes": {
         "digital_interrupts": [
-          {
-            "name": "<your-digital-interrupt-name>",
-            "pin": "<pin-number>"
-          }
+          <...See table below...>
```
[the remark this answers is not in the document]
Comment (sguequierre) [reply docs/components/board/upboard.md:54]: re-added -- I did think that that made sense as table directs to template and instructions that come from the include below?

[the remark this answers is not in the document]
Comment (sguequierre) [reply docs/components/board/upboard.md:54]: But can't hurt to have it here as well

File: docs/components/board/numato.md [hunk 3]
```diff
@@ -25,29 +25,7 @@ Enter a name for your board and click **Create**.
 
 ![An example configuration for a numato board in the Viam app Config Builder.](/components/board/numato-ui-config.png)
 
-Copy and paste the following attribute template into your board's **Attributes** box.
```
[the remark this answers is not in the document]
Comment (sguequierre) [reply docs/components/board/numato.md:28]: Is 43-52 below what you meant? @npentrel

File: static/include/components/board-attr-config.md [hunk 4]
```diff
@@ -1,37 +0,0 @@
-Copy and paste the following attribute template into your board's **Attributes** box.
```
[the remark this answers is not in the document]
Comment (sguequierre) [reply static/include/components/board-attr-config.md:1]: Should all be in analogs and/or digital interrupts includes!

File: docs/components/board/numato.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (npentrel) [reply docs/components/board/numato.md:28]: nice yeah thanks!

File: docs/components/board/beaglebone.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (npentrel) [reply docs/components/board/beaglebone.md:31]: I think this is good

**Pull request merged**